src/overlays/actors/ovl_En_Fg/z_en_fg.c

```diff
@@ -369,7 +369,7 @@ void EnFg_Update(Actor* thisx, PlayState* play) {
     flagSet = CHECK_FLAG_ALL(flag, ACTOR_FLAG_2000);
     if (1) {}
     if (!flagSet) {
-        flagSet = CHECK_FLAG_ALL(flag, ACTOR_FLAG_8000);
+        flagSet = CHECK_FLAG_ALL(flag, ACTOR_FLAG_ATTACHED_TO_ARROW);
```

Comment:
Not sure if better/should be done in this PR, but you can remove the temps and `if (1)`s by doing 
```c
    if ((CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_2000) == 0) &&
        (CHECK_FLAG_ALL(this->actor.flags, ACTOR_FLAG_8000) == 0)) {
        this->actionFunc(this, play);
        Actor_UpdateBgCheckInfo(play, &this->actor, sREG(0), sREG(1), 0.0f,
                                UPDBGCHECKINFO_FLAG_1 | UPDBGCHECKINFO_FLAG_4);
    }
```

Reply:
Nice find!